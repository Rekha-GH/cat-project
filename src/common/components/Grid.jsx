import React from "react";

const DEFAULT_CLASSNAME = "grid";
const Grid = React.memo((props) => {
  return <section className={DEFAULT_CLASSNAME}>{props.children}</section>;
});

export default Grid;
