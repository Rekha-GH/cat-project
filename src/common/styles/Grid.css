.grid {
  padding: 1px;
  display: flex;
  flex-wrap: wrap;
  gap: 15px;
}
