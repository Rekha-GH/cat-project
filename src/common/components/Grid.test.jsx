import Enzyme, { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "enzyme-adapter-react-16";

import Grid from "./Grid";

Enzyme.configure({ adapter: new Adapter() });

describe("Grid Test", () => {
  test("Grid Call Correctly", () => {
    const wrapper = shallow(<Grid />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
