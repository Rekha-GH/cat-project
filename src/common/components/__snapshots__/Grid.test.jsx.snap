// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Test Grid Call Correctly 1`] = `
<section
  className="grid"
/>
`;
